src/main/java/com/pingcap/tikv/policy/RetryNTimes.java

```diff
@@ -17,35 +17,29 @@
 
 import com.google.common.base.Preconditions;
 import com.pingcap.tikv.operation.ErrorHandler;
+import com.pingcap.tikv.util.ExponentialBackOff;
 
-public class RetryNTimes extends RetryPolicy {
-  private int n;
-
-  private RetryNTimes(int n, ErrorHandler handler) {
+public class RetryNTimes<T> extends RetryPolicy<T> {
+  private RetryNTimes(int n, ErrorHandler<T> handler) {
     super(handler);
+    this.backOff = new ExponentialBackOff(3);
```

Comment:
If backoff is decoupled, this should be decoupled here as well. Lift this up to PolicyBuilder

Reply:
Done.